src/servers/src/context.rs

```diff
@@ -80,19 +81,20 @@ impl CtxBuilder {
     }
 }
 
+#[derive(Clone)]
 pub struct ClientInfo {
     pub client_host: String,
     pub channel: Channel,
 }
 
-#[derive(Debug, PartialEq, Eq)]
+#[derive(Clone, Debug, PartialEq, Eq)]
 pub enum Channel {
     GRPC,
     HTTP,
     MYSQL,
```

Comment:
It seems that Rust tends to use CamelCase for Enum
https://doc.rust-lang.org/1.0.0/style/style/naming/README.html